src/pineko/cli/check.py

```diff
@@ -28,3 +28,33 @@ def subcommand(pineappl_path, eko_path, xif):
         rich.print("[green]Success:[/] grids are compatible")
     except ValueError as e:
         rich.print("[red]Error:[/]", e)
+
+
+@command.command("check_scalevar")
+@click.argument("pineappl_path", metavar="PINEAPPL", type=click.Path(exists=True))
+@click.argument(
+    "tocheck",
+    metavar="TOCHECK",
+    type=str,
+)
+def subcommand_sv(pineappl_path, tocheck):
```

Comment:
This one I want to try to refactor myself, I'll publish a candidate as soon as I have time.